pkg/kubefed2/federate/enable.go

```diff
@@ -124,32 +131,56 @@ func NewCmdFederateEnable(cmdOut io.Writer, config util.FedConfig) *cobra.Comman
 
 // Complete ensures that options are valid and marshals them if necessary.
 func (j *enableType) Complete(args []string) error {
+	j.federateDirective = &FederateDirective{}
+	ft := j.federateDirective
```

Comment:
Not nitpicking on naming, but `fd` seems more apt here. I saw `ft` in later section of code and wondered what might it be.

Reply:
Good catch.  I had originally named the type `FederateType`.